bundle/tests/scorecard/config.yaml

```diff
@@ -0,0 +1,29 @@
+kind: Configuration
+apiversion: scorecard.operatorframework.io/v1alpha3
+metadata:
+  name: config
+
+stages:
+  - tests:
+      - image: quay.io/ytripath/cert-manager-operator-e2e-tests:latest
+        entrypoint:
+          - custom-scorecard-tests
+          - TestSelfSignedCerts
+        labels:
+          suite: custom
+          test: TestSelfSignedCerts
+      - image: quay.io/ytripath/cert-manager-operator-e2e-tests:latest
+        entrypoint:
+          - custom-scorecard-tests
+          - TestACMECertsIngress
+        labels:
+          suite: custom
+          test: TestACMECertsIngress
+      - image: quay.io/ytripath/cert-manager-operator-e2e-tests:latest
+        entrypoint:
+          - custom-scorecard-tests
+          - TestCertRenewal
```

Comment:
Same question as https://github.com/openshift/cert-manager-operator/pull/64/files#r871078707 . After https://github.com/openshift/cert-manager-operator/pull/64/files#r871078707 is solved, maybe we need not enumerate scenarios under "- tests:" as well and only one single item under "- tests:" is enough?